src/controllers/course.ts

```diff
@@ -1,87 +1,75 @@
-import { Request, Response} from 'express';
-import {AppDataSource} from "../config/data-source"
-import {Course} from "../entity/course" 
+import { Request, Response } from 'express';
+import Course from "../model/course";
 
-class CourseController{
-    public static async index(req: Request, res: Response){
-        const course = await AppDataSource.manager.find(Course)
-        return res.status(200).json({
-            message: course
-        });  
+class CourseController {
+    public static async index(req: Request, res: Response) {
```

Comment:
we should avoid using static fo methods unless absolutely required. instead it's preferable to create the instance of the class or use Singleton pattern instead

Reply:
Done!